commit 88f52ebcac047350724b4947f776d1910ee74c0f
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-06-19T21:57:53+00:00 (4 months ago)
    
    MANTA-4305 want a waferlock that uses pg_hba.conf instead of ipfilter
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index 926ae76..facaadb 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -34,6 +34,7 @@ function manta_manatee_setup {
 
     echo "Adding local manifest directories"
     manta_add_manifest_dir "/opt/smartdc/manatee"
+    manta_add_manifest_dir "/opt/smartdc/waferlock"
 
     # MANTA-1360 no args to manta_common_setup so we don't rotate the 'manatee'
     # entry which stomps over the other manatee logs
@@ -81,6 +82,9 @@ function common_enable_services {
     #
     echo "Starting prefaulter"
     svccfg import /opt/smartdc/manatee/smf/manifests/pg_prefaulter.xml
+
+    echo "Starting waferlock"
+    svccfg import /opt/smartdc/waferlock/smf/manifests/waferlock.xml
 }
 
 function common_manatee_setup {
@@ -166,6 +170,7 @@ function manta_setup_manatee_env {
     manta_add_logadm_entry "manatee-sitter"
     manta_add_logadm_entry "manatee-backupserver"
     manta_add_logadm_entry "manatee-snapshotter"
+    manta_add_logadm_entry "waferlock"
     manta_add_logadm_entry "postgresql" "/var/pg"
     manta_add_logadm_entry "pgdump" "/var/log/manatee"
 }
